src/status_im/ui/components/list_item/views.cljs

```diff
@@ -245,114 +247,144 @@
-;; i.e. when icon height/height > 16, or when component
-;; do not specify if title-prefix is tiny-icon
-
-;; title
-;; any one of string, keyword representing translated string in the form of
-;; :t/{translation-key-in-translation-files},
-;; keyword(gets converted to string),
-;; number(gets converted to string), or
-;; component - when component is used best to keep the style similar
-;; to `styles/title-row-container` and/or `styles/title`
-
-;; title-color-override
-;; colors/color - only occasionally needed, self-explanatory
-
-;; title-accessibility-label
-;; :accessibility-label for title text component
-;; sometimes needed for title - e.g. chat-list-item
-;; makes sense since `title` is the key element of a list item
-
-;; title-row-accessory
-;; component - especially made for chat list item, but may serve other
-;; purpose in the unlikely future. Wrapper already has 2px :margin-top
-;; best to keep it <= 18px high
-
-;; subtitle
-;; any one of string, keyword representing translated string in the form of
-;; :t/{translation-key-in-translation-files},
-;; keyword(gets converted to string),
-;; number(gets converted to string), or
-;; component - when component is used best to keep the style similar
-;; to `styles/subtitle-title-row-container` and/or `styles/subtitle`
-
-;; subtitle-max-lines
-;; integer - 1 by default - self-explanatory
-
-;; subtitle-row-accessory
-;; component
-;; made specially for chat-list to hold unread messages counter
-
-;; content
-;; component - to replace entire title-column area
-;; TODO - consider removing, as it is redundant now that
-;; title/subtitle elements can be component as well
-;; just make sure to keep in mind the note made on
-;; component case on those keys above
-
-;; accessories
-;; vector of :chevron, :check, :more, string, number, keyword or component
-
-;; on-press/on-long-press
-;; function - self explanatory
-
-;; error
-;; string - error tooltip
-
-;; accessibility-label
-;; :keyword - self explanatory
-
-;; disabled?
-;; boolean - false by default - self explanatory
-
 (defn list-item
-  [{:keys [type theme container-margin-top container-margin-bottom
-           icon title-prefix title-prefix-width title-prefix-height
-           title title-color-override title-row-accessory
-           title-accessibility-label subtitle subtitle-max-lines
-           subtitle-row-accessory content accessories on-press
-           on-long-press error accessibility-label disabled?]
-    :or   {type                    :default
-           theme                   :default
-           disabled?               false
-           container-margin-top    0
-           container-margin-bottom 0
-           subtitle-max-lines      1}}]
+  "A general purpose Status-React specfic list item component.
+  Every key is optional. Use as needed.
+  Combination of around 4 related keys are enough for most cases.
+  Spec: https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=787%3A1108
+
+  `react-key`
```

Comment:
is it used somewhere? i believe we use metadata for that `^{:key ""}`

Reply:
Adding this is indeed a nitpick in a way :) Reasoning for having it in list-item component is similar to why `flat-list` component has a `keyExtractor` prop https://facebook.github.io/react-native/docs/flatlist#keyextractor.  Plus with this, we can even ignore it, since there is a sane default with `gensym`. And one less nesting of form in instances where  it is really needed.